src/mame/mame.lst

```diff
@@ -19144,6 +19144,7 @@ h19                             // Heath H19 (Zenith Z-19)
 super19                         // Super19 replacement ROMS for H19
 watz19                          // Watzman replacement ROMS for H19
 ultra19                         // ULTRA ROM replacement ROMS for H19
+gp19                            // H19 with Northwest Digital Systems GP-19
```

Comment:
Please keep systems sorted alphabetically within a source file group.

Reply:
I fixed the ultra/watz order when I fixed it for the new gp19. I wasn't sure if the parent system should always be at the top or also alphabetized, so I just left that as the first one. I looked at a few others, and didn't see anything conclusive. Let me know if I should change that.